frontend/src/renderer/components/Process/Tabs/ColorTarget.svelte

```diff
@@ -238,22 +238,18 @@
   $: console.log({ LOADING: loading });
 
   $: if (
-    colorTarget != null &&
-    !$processState.completedTabs[4] &&
+    colorTarget &&
     colorTarget.refData.name !== "---None---.csv" &&
-    (verifyTarget != null
-      ? verifyTarget.refData.name !== "---None---.csv"
-      : true) &&
-    colorTarget.whitePatch.row != null &&
-    colorTarget.whitePatch.col != null
+    (!verifyTarget || verifyTarget.refData?.name !== "---None---.csv") &&
+    colorTarget?.whitePatch?.row && colorTarget?.whitePatch?.col
```

Comment:
I believe the issue was in this above condition (although I cleaned up some other related code). This condition is almost exactly the same logic, with a removal of line 242 (which could be related to the issue?) and some better null-handling.

Reply:
I do think this is better than what it was but I am concerned with removing "!$processState.completedTabs[4]".  I tested it on my machine and everything works.  I am thinking this could cause a problem when merging the new front in.